pkgs/applications/misc/navit/default.nix

```diff
@@ -1,17 +1,27 @@
-{ stdenv, fetchFromGitHub, pkgconfig, gtk2, SDL, fontconfig, freetype, imlib2, SDL_image, libGLU_combined,
-libXmu, freeglut, pcre, dbus-glib, glib, librsvg, freeimage, libxslt,
-qtbase, qtquickcontrols, qtsvg, qtdeclarative, qtlocation, qtsensors, qtmultimedia, qtspeech, espeak,
-cairo, gdk_pixbuf, pango, atk, patchelf, fetchurl, bzip2,
-python, gettext, quesoglc, gd, postgresql, cmake, shapelib, SDL_ttf, fribidi}:
+{ stdenv, fetchFromGitHub, pkgconfig, gtk2, fontconfig, freetype, imlib2
+, SDL_image, libGLU_combined, libXmu, freeglut, pcre, dbus, dbus-glib, glib
+, librsvg, freeimage, libxslt, cairo, gdk_pixbuf, pango
+, atk, patchelf, fetchurl, bzip2, xkbd, python, gettext, quesoglc
+, gd, cmake, shapelib, SDL_ttf, fribidi, espeak, makeWrapper
+, qtquickcontrols, qtmultimedia, qtspeech, qtsensors
+, qtlocation, qtdeclarative, qtsvg
+, qtSupport ? false, qtbase #need to fix qt_qpainter
+, sdlSupport ? true, SDL
+, postgresqlSupport ? false, postgresql
+, speechdSupport ? false, speechd ? null
+}:
 
+assert speechdSupport -> speechd != null;
+
+with stdenv.lib;
 stdenv.mkDerivation rec {
   name = "navit-${version}";
   version = "0.5.1";
 
   src = fetchFromGitHub {
     owner = "navit-gps";
     repo = "navit";
-    rev = "v${version}";
+    rev = version;
```

Comment:
Why did you change this? Looks like they do tag their releases with `v${version}`: https://github.com/navit-gps/navit/releases

Reply:
oh my bad!